examples/src/main/res/drawable/user_puck_icon.xml

```diff
@@ -12,6 +12,6 @@
         android:fillColor="#FFFFFF"
         android:pathData="M37.5,37.5m-28.5,0a28.5,28.5 0,1 1,57 0a28.5,28.5 0,1 1,-57 0"/>
     <path
-        android:fillColor="@color/jungleAccentOrange"
+        android:fillColor="@color/accent_orange"
```

Comment:
`user_puck_icon.xml` is not being used anymore from the `examples` test app. Could we remove? One thing that would be cool though is to change it (e.g. the color so it doesn't match the default one) and showcase how it can be customized. What do you think?

Reply:
We could add an example activity to demonstrate this along with the other example activities.